tests/h/traversal/roots_test.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+import mock
+
+from h.models import AuthClient
+from h.services.user import UserService
+from h.traversal.roots import AnnotationRoot
+from h.traversal.roots import AuthClientRoot
+from h.traversal.roots import OrganizationRoot
+from h.traversal.roots import OrganizationLogoRoot
+from h.traversal.roots import GroupRoot
+from h.traversal.roots import UserRoot
+from h.traversal.contexts import AnnotationContext
+
+
+@pytest.mark.usefixtures('group_service', 'links_service')
+class TestAnnotationRoot(object):
+    def test_get_item_fetches_annotation(self, pyramid_request, storage):
```

Comment:
Would it be useful to append `_resource` to this test method's name? I spent a bit of time wondering until I saw the following test that was explicit about it.